fluss-lake/fluss-lake-paimon/src/test/java/com/alibaba/fluss/lake/paimon/flink/FlinkUnionReadPrimaryKeyTableITCase.java

```diff
@@ -259,6 +260,168 @@ void testUnionReadWithTimeStamp() throws Exception {
                         "[+I[1, 2023-10-25T12:01:13.182005Z, 2023-10-25T12:01:13.183006], +I[2, 2023-10-25T12:01:13.400007Z, 2023-10-25T12:01:13.501008]]");
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {false, true})
+    void testUnionReadFullType(Boolean isPartitioned) throws Exception {
```

Comment:
All tests in this class are pretty heavy as they need mini-cluster resources, we can reduce `testUnionReadFullType` and `testPrimaryKeyTable` to one for example `testPrimaryKeyTableWithFullTypes` to save the test cost, and  `testUnionReadWithTimeStamp` can be improve the `testUnionReadWithTimeStampAndFullTypes`. In this way, we improve our test coverage but not waste unnecessary test cost.